style.css

```diff
@@ -0,0 +1,142 @@
+html {
+    font-size: 10px;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+}
+
+h1 {
+    color: #ffffff;
+    font-weight: 700;
+    font-size: 10rem;
+    text-align: center;
+}
+
+h2 {
+    font-size: 7rem;
+}
+
+p {
+    font-size: 1.8rem;
+}
+
+.nav-bar {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    color: #ff8000;
+    font-size: 1.2rem;
+    text-transform: uppercase;
+    font-weight: 400;
+}
+
+.nav-bar a:link, .nav-bar a:visited {
+    color: #ff8000;
+    text-decoration: none;
+}
+
+.nav-bar a:hover {
+    color: #001433;
+    text-decoration: underline;
+}
+
+.nav-menu {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+
+}
+
+.nav-menu-item {
+    margin: 0.2em;
+    padding: 0.6em;
+}
+
+.nav-title {
+    font-size: 2.5rem;
+    font-weight: 700;
+    letter-spacing: 0.15rem;
+}
+
+header {
+    background-color: #ff8000;
+    padding-top: 25px;
+    padding-bottom: 25px;
+    min-height: 60vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.intro-section {
```

Comment:
needs some left and right padding or margin - if sized right before wrapping width it ends up with no space on the sides (where you probably want space)
![image](https://user-images.githubusercontent.com/25844355/149813860-ce134b89-1d50-49b5-9ab0-05402bb2d0fa.png)